pkg/remote/oci/resolver.go

```diff
@@ -29,16 +33,30 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
-var _ remote.Resolver = (*Resolver)(nil)
+const (
+	KindAnnotation       = "dev.tekton.image.kind"
+	APIVersionAnnotation = "dev.tekton.image.apiVersion"
+	TitleAnnotation      = "dev.tekton.image.name"
+	MaximumBundleObjects = 10
```

Comment:
Is it documented somewhere that we limit to 10 the content of a bundle ?

Reply:
Yup. It is spelled out in the contract: https://github.com/tektoncd/pipeline/pull/3142/files#diff-1dfbd644a9243c47edaba1835ff18e89bde7cb3b25b475356d5827d1bf8bbea9R20

Reply:
Ah right, make sense. We can change this later on if we need :+1: